src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Diagnostics/DocumentPullDiagnosticsEndpoint.cs

```diff
@@ -25,6 +28,14 @@ internal class DocumentPullDiagnosticsEndpoint : IRazorRequestHandler<VSInternal
     private readonly RazorTranslateDiagnosticsService _translateDiagnosticsService;
     private readonly ITelemetryReporter? _telemetryReporter;
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="DocumentPullDiagnosticsEndpoint"/> class.
+    /// </summary>
+    /// <param name="languageServerFeatureOptions">The <see cref="LanguageServerFeatureOptions"/>.</param>
+    /// <param name="translateDiagnosticsService">The <see cref="RazorTranslateDiagnosticsService"/>.</param>
+    /// <param name="languageServer">The <see cref="ClientNotifierServiceBase"/>.</param>
+    /// <param name="telemetryReporter">The <see cref="ITelemetryReporter"/>.</param>
+    /// <exception cref="ArgumentNullException"/>
```

Comment:
Nit: I'd rather not have the visual noise in the file if the docs don't add any value.

```suggestion
```